roles/slurm/handlers/main.yml

```diff
@@ -10,11 +10,19 @@
 - name: 'Restart munge service.'
   ansible.builtin.systemd:
     name: 'munge.service'
-    state: restarted
+    state: 'restarted'
     daemon_reload: true
   become: true
   listen: restart_munge
 
+- name: 'Restart slurmd service.'
+  ansible.builtin.systemd:
+    name: 'slurmd.service'
+    state: 'restarted'
+    daemon_reload: true
+  become: true
+  listen: restart_slurmd
+
```

Comment:
Handlers run in the order specified in this file. Order should be:

1. slurmdbd
2. slurmctld
3. slurmd

Applies to both restarts and reloads also in that order.

Reply:
Yes. Indeed - good call. I'll change it!